src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -3,18 +3,150 @@
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.Extensions.Primitives;
+using Raven.Client.Exceptions.Database;
+using Raven.Client.Server;
 using Raven.Server.Documents.Handlers.Admin;
+using Raven.Server.Exceptions;
+using Raven.Server.Json;
 using Raven.Server.Routing;
 using Raven.Server.ServerWide;
 using Raven.Server.ServerWide.Context;
 using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Logging;
 
 namespace Raven.Server.Documents.Handlers.Debugging
 {
     public class ServerWideDebugInfoPackageHandler : AdminRequestHandler
     {
+        private static readonly string[] EmptyStringArray = new string[0];
+        private static readonly Logger _log = LoggingSource.Instance.GetLogger<ServerStore>(nameof(ServerWideDebugInfoPackageHandler));
+
+        //this endpoint is intended to be called by /debug/cluster-info-package only
+        [RavenAction("/debug/remote-cluster-info-package", "GET")]
+        public async Task GetClusterwideInfoPackageForRemote()
+        {
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext transactionOperationContext))
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext jsonOperationContext))
+            using (transactionOperationContext.OpenReadTransaction())
+            {
+                using (var ms = new MemoryStream())
+                {
+                    using (var archive = new ZipArchive(ms, ZipArchiveMode.Create, true))
+                    {
+                        var localEndpointClient = new LocalEndpointClient(Server);
+                        NodeDebugInfoRequestHeader requestHeader;
+                        using (var requestHeaderJson = await transactionOperationContext.ReadForMemoryAsync(HttpContext.Request.Body, "remote-cluster-info-package/read request header"))
+                        {
+                            requestHeader = JsonDeserializationServer.NodeDebugInfoRequestHeader(requestHeaderJson);
+                        }
+
+                        await WriteServerWide(archive, jsonOperationContext, localEndpointClient);
+                        foreach (var databaseName in requestHeader.DatabaseNames)
+                        {
+                            await WriteForDatabase(archive, jsonOperationContext, localEndpointClient, databaseName);
+                        }
+                    }
+
+                    ms.Position = 0;
+                    await ms.CopyToAsync(ResponseBodyStream());
+                }
+            }
+        }
+
+        [RavenAction("/debug/cluster-info-package", "GET", IsDebugInformationEndpoint = true)]
+        public async Task GetClusterwideInfoPackage()
+        {
+            var contentDisposition = $"attachment; filename=Cluster wide debug-info {DateTime.UtcNow}.zip";
+            HttpContext.Response.Headers["Content-Disposition"] = contentDisposition;
+
+            using (var httpClient = new HttpClient())
```

Comment:
Wrong. You cannot do this.
We have the `RequestExecuter` for a reason. To start with, you are not making the call using the appropriate credentials, so this will never work with authenticated setups.